app/_src/deck/reference/entities.md

```diff
@@ -26,14 +26,12 @@ Workspaces | <i class="fa fa-check"></i> <sup>2</sup>
 RBAC: roles and endpoint permissions | <i class="fa fa-check"></i>
 RBAC: groups and admins | <i class="fa fa-times"></i>
 Developers | <i class="fa fa-times"></i>
-
-{% if_version lte:1.16.x inline:true %}
+{% if_version lte:1.16.x -%}
 Consumer groups | <i class="fa fa-times"></i>
-{% endif_version %}
-{% if_version gte:1.17.x inline:true %}
+{% endif_version -%}
+{% if_version gte:1.17.x -%}
```

Comment:
For adjacent versions like this, where something changed in a given release and we have `lte` and `gte` next to each other, should we use `else` instead? Or would you rather not start converting that in this PR?

Reply:
yeah, I would leave them out for now, and refactor them as we go whenever we find one.